### ISSUE TITLE
Add UTF-8 encoding

### ISSUE THREAD

**Issue by stephparker** [NEONScience/NEON-utilities#54]
**Function**
loadByProduct

**Describe the bug**
Trying to load algae taxonomy data, special characters in scientificNames and other fields load with many messed up characters. Eric reproduced the issue as well.

**To Reproduce**
alg_allTabs <- loadByProduct(dpID = "DP1.20166.001", 
  site = "all", package = "expanded", 
  check.size = FALSE))
list2env(alg_allTabs,envir=.GlobalEnv) 
see alg_taxonomyProcessed table

**Expected behavior**
Pull data using loadByProduct or other calls and get text with correctly encoded special characters

**System (please complete the following information):**
 - OS: Windows 
 - OS Version: 10
 - R Version: 3.5.3

**Additional context**
Eric thinks that if we add encodiing = UTF-8 for the data.table::fread call in stackeddatafiles it might fix the problem.

**Comment by cklunch** [comment]
Thanks for the heads up! This is now fixed in the rhdf5-suggests branch.
@stephparker can you install the branch version and verify this is working? If you haven't installed from a branch before, it's like this:
```
devtools::install_github("NEONScience/NEON-utilities/neonUtilities", ref="rhdf5-suggests")
```

**Comment by stephparker** [comment]
@cklunch I loaded the branch but loadByProduct won't run. 

Error in loadByProduct(dpID = "DP1.20166.001", site = "all", package = "expanded",  : 
  could not find function "loadByProduct"

**Comment by cklunch** [comment]
Resolved in neonUtilities v1.3.1.
